doc/src/records/rsys.md

```diff
@@ -136,6 +136,10 @@ Specifies type of motor driving the fan (blower). This is used in the derivation
   ----------- ----------------- ------------- -------------- -----------------
               PSC, BPM          PSC           No             constant
 
+**rsAdjForFanHt=*NOYESCH***
```

Comment:
I think this is just ***choice***. Look for other examples. I could be wrong.

Reply:
I think I saw other examples like that. If I see them again I'll change them to choice.